src/snowflake/cli/api/project/schemas/native_app/path_maping.py

```diff
@@ -0,0 +1,10 @@
+from __future__ import annotations
```

Comment:
This filename has a typo (should be `path_mapping.py`).

Reply:
Done